org.nongnu.gsequencer.gsequencer.json

```diff
@@ -212,15 +254,32 @@
                 }
             },
             "post-install": [
+                "mkdir -p /app/share/doc/gsequencer/pdf/",
+                "install -c -p -m 644 pdf/user-manual-a4.pdf /app/share/doc/gsequencer/pdf/user-manual-a4.pdf",
+                "install -c -p -m 644 pdf/user-manual-letter.pdf /app/share/doc/gsequencer/pdf/user-manual-letter.pdf",
                 "sed -i -e 's~Icon=gsequencer~Icon=org.nongnu.gsequencer.gsequencer~g' /app/share/applications/gsequencer.desktop",
                 "install -c -p -m 644 COPYING /app/share/gsequencer/GPL-3",
                 "install -d /app/extensions/Plugins"
             ],
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download-mirror.savannah.gnu.org/releases/gsequencer/4.0.x/gsequencer-4.0.13.tar.gz",
-                    "sha256": "dba48feed9e8455c5234f002cc874bfb55e51376024046ee5faa4ed528664084"
+                    "url": "https://download-mirror.savannah.gnu.org/releases/gsequencer/4.1.x/gsequencer-4.1.1.tar.gz",
+                    "sha256": "1dbda4450fb4744d7f45acd80b8f5aff53f849f196b3343d97f1cb1a65e0c991"
+                },
+                {
+                    "type": "file",
+                    "url": "http://xn--krhemann-1za.com/download/ags-user-manual-a4.pdf",
```

Comment:
I hope you don't plan to update it without changing the name.

Reply:
I am unsure what do you want?

Reply:
if you regenerate the manual (like for a future release) its sha256 will change and the build will be broken. This is what happen with non-versioned resources. tl;dr new revision = new name, and there is no indication of that here.

(this apply to the other too)

Reply:
Just provide PDFs with version information.

Reply:
just added version information to filenames.